src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+  let trueRobot = {};
+
+  for (const key in robot) {
+    if (trueRobot[robot[key]]) {
+      trueRobot = null;
```

Comment:
[CODE STYLE] - use correct check for property presence in object. Some properties can be present but still contain falsy value.
you chose a bad example with an uncleared solution.